istio-operator/istio-operator.spec

```diff
@@ -13,7 +13,7 @@
 %global debug_package   %{nil}
 %endif
 
-%global git_commit 259bb0321cdabf58218190ef0ac8bf57eeaae43e
+%global git_commit 3a076c29531c848e5d2e846832eb33717f4be042
```

Comment:
should this be ad08ae640ef46cf7a3a25229329e1e02570691a4 ?

Reply:
This pr was issued before an accidental pr on operator got merged. This morning Daniel pushed another one fixing it, that I just picked up.